extensions/src/AWSSDK.Extensions.NETCore.Setup/ClientFactory.cs

```diff
@@ -367,9 +364,27 @@ private ClientConfig CreateConfig(AWSOptions options)
                 ProcessServiceSpecificSettings(config, defaultConfig.ServiceSpecificSettings);
             }
 
+            // It is possible that this library might be used in .NET Framework application. In that
+            // case the SDK used at runtime will be the .NET Framework variant. To avoid getting
+            // MissingMethodException exceptions for config properties that don't exist in 
+            // .NET Framework variant put the set calls in a separate method only called
+            // if the SDK is not the .NET Framework variant.
+            if (!AWSSDKUtils.IsNETFramework())
+            {
+                SetNETStandardAndAboveSettings(defaultConfig, config);
+            }
+
             return config;
         }
 
+        private void SetNETStandardAndAboveSettings(DefaultClientConfig defaultConfig, ClientConfig config)
```

Comment:
To be extra sure that the non-existent properties don't get resolved, this method should be marked with `[MethodImpl(MethodImplOptions.NoInlining)]`.

Reply:
Done, thanks for the suggestion.